src/main/java/com/aphyr/riemann/client/Callback.java

```diff
@@ -0,0 +1,7 @@
+package com.aphyr.riemann.client;
+
+public interface Callback<T> {
+
+  public void call(T value, Exception exception);
```

Comment:
Sure. I'm pretty indifferent to one-callback-could-be-a-value-or-exception, two callbacks, or one-callback-two-different-values.